Policies/Kubernetes/container-required-probes/azurepolicy.json

```diff
@@ -0,0 +1,63 @@
+{
+  "properties": {
+    "displayName": "Ensure all pods have lifecycle probes configured",
+    "policyType": "BuiltIn",
+    "mode": "Microsoft.Kubernetes.Data",
+    "description": "This policy enforces that all pods have the expected lifecycle probes configured. This policy is generally available for Kubernetes Service (AKS), and preview for AKS Engine and Azure Arc enabled Kubernetes. For instructions on using this policy, visit https://aka.ms/kubepolicydoc.",
+    "metadata": {
```

Comment:
Remove text "This policy is generally available for Kubernetes Service (AKS), and preview for AKS Engine and Azure Arc enabled Kubernetes" as we release this policy initially in preview mode.